generator/graphql-kotlin-federation/src/test/kotlin/com/expediagroup/graphql/generator/federation/validation/integration/FederatedKeyDirectiveIT.kt

```diff
@@ -128,7 +129,7 @@ class FederatedKeyDirectiveIT {
         assertEquals(expected, exception.message)
     }
 
-    @Test
+    @Disabled("This is a valid case according to Federation 2 spec")
```

Comment:
instead of disabling this test, this code (and referenced sample project) should be deleted